R/case.R

```diff
@@ -1,21 +1,41 @@
-#' Convert string to upper case, lower case, title case, or sentence case
+#' Convert string to upper case, lower case, title case, sentence case, pascal
```

Comment:
Maybe it would be better to document these new functions in a new documentation topic? Yes, they're are changing case, but you're more likely to be apply them to function names, not regular sentences.